flutter/test/default_integrations_test.dart

```diff
@@ -46,6 +46,9 @@ void main() {
     final details = FlutterErrorDetails(
       exception: throwable,
       silent: silent,
+      context: DiagnosticsNode.message('while handling a gesture'),
+      library: 'sentry',
+      informationCollector: () => [DiagnosticsNode.message('foo bar')],
```

Comment:
Worth adding a test to validate the behavior with multiple lines (since this is an array).
Also good to check what it looks like in Sentry.